.circleci/config.yml

```diff
@@ -300,8 +300,8 @@ jobs:
             set -exu
             mkdir -p /tmp/workspace/js
             mkdir -p /tmp/workspace/css
-            mv /tmp/workspace/js/* jekyll/assets/js/
-            mv /tmp/workspace/css/* jekyll/assets/css/
+            mv -fn /tmp/workspace/js/* jekyll/assets/js/
+            mv -fn /tmp/workspace/css/* jekyll/assets/css/
```

Comment:
`-fn`: `f` forces and `n` doesn't overwrite. CI was giving us trouble without that